components/elm/src/biogeophys/SedYieldMod.F90

```diff
@@ -225,9 +314,9 @@ character(len=32) function SoilTextureType(stxt)
     !------------------------------------------------------------------------------
 
     tsum = sum(stxt(1:3))
-    clay = stxt(1) / tsum
-    silt = stxt(2) / tsum
-    sand = stxt(3) / tsum
+    clay = 1.e2_r8 * stxt(1) / tsum 
```

Comment:
add a comment about this `1.e2_r8` factor. It would be even better if you declared this as a parameter in the beginning of the subroutine.